src/pages/Login/index.tsx

```diff
@@ -56,21 +48,18 @@ const Login = () => {
   }
 
   const submitHandler = (e: React.FormEvent) => {
+    const token = LocalStorage.getToken()
```

Comment:
I think you don't need to check for the token. If there's no token or token is invalid, the saga will receive request error and you can display the error from the saga

Reply:
True, it works without token. I removed it.